pulldown-cmark/src/parse.rs

```diff
@@ -62,6 +62,8 @@ pub(crate) enum ItemBody {
 
     // repeats, can_open, can_close
     MaybeEmphasis(usize, bool, bool),
+    // is_display, can_open, can_close, brace context
```

Comment:
It's tempting to replicate the style of the surrounding code here. Maybe we should change all of them to use named parameters.

Reply:
I have tried to change that but sadly it makes dirtier other parts of the code, where it is clear the meaning of each field because there they have names, so I have just removed the first wrong field of the comment.